CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # TFS-Pandas Changelog
 
+## Version 3.7.1
+
+- Changed:
+    - The dependency on `pandas` was restricted to avoid the latest version (and above, for now) as a temporary workaround to an attribute access bug that arose with it. 
```

Comment:
write explicitly that version 2.1.0 does not work

Reply:
done